src/Tracy/Debugger/Debugger.php

```diff
@@ -323,7 +323,9 @@ public static function exceptionHandler(\Throwable $exception): void
 				@fwrite(STDERR, "ERROR: {$exception->getMessage()}\n"
 					. (isset($e)
 						? 'Unable to log error. You may try enable debug mode to inspect the problem.'
-						: 'Check log to see more info.')
+						: 'Check log to see more info.') . "\n\n"
```

Comment:
To by si mělo vypisovat spíš jen v debug režimu, ne?

Reply:
CLI je vždy produkční, ne?

Reply:
Ne, je to nezávislé.

Reply:
Doplněno.